### ISSUE TITLE
Mobile?

### ISSUE THREAD

**Issue by Trey336** [Windblade-GR01/Ikemen-GO#343]
Is making a mobile port possible? I mean, atleast I asked, but if it is impossible, I'm sorry for wasting everyone's precious time, ikemen on the go would be awesome. Has anyone tried to do this? 🤔

**Comment by Adnan1091** [comment]
This was asked several times(including me).The author says GLFW doesn't have mobile support,but i saw it was discussed in GLFW's github page and a fork that supports android is avilable as far as i know.

**Comment by xx1182** [comment]
If you really, really want... and don't mind being autistic:
You can use Termux setup a VNCServer and boom, it works.

First you gotta compile tho.

Regards.

**Comment by XTRAXTREME** [comment]
Someone explain me this

**Comment by xx1182** [comment]
> Someone explain me this
Sure, let's exchange addresses, or do it in videocall, is a bit complicated depending on your expertise in programming and troubleshooting, because it inquires:
Setting up an environment to compile your binary


> Someone explain me this
Sure but then you write a wiki article

**Comment by DJPilouDevelopersandmore** [comment]
i do this ported by android and ios mobile

**Comment by Orochikyocr** [comment]
This should be closed, because tbh, once Gacel feels the engine is ready to go Android, he will not hesitate and it has been asked so many times.

**Comment by Windblade-GR01** [comment]
Closing because is a duplicate of #224.
